src/xo-app/vm/tab-general.js

```diff
@@ -59,7 +66,11 @@ export default ({
         ? <div>
           <p className='text-xs-center'>{_('started', { ago: <FormattedRelative value={vm.startTime * 1000} /> })}</p>
         </div>
-        : <p className='text-xs-center'>{_('vmNotRunning')}</p>
+        : <p className='text-xs-center'>
+          { lastShutdownTime
+            ? _('vmNotRunningHaltedFor', {ago: <FormattedRelative value={lastShutdownTime * 1000} />})
+            : _('vmNotRunning')}
```

Comment:
Closing brace should be on a line with the same indentation level as the opening brace.